src/App.tsx

```diff
@@ -6,46 +6,76 @@ import { usePersistentState } from './PersistentState';
 import { useState, useEffect } from 'react';
 import './App.css';
 
+// truncate number to n decimal places
+const trunc = (numberToTruncate: number, numberOfDecimalPlaces: number) => {
+  const tenexp = 10 ** numberOfDecimalPlaces;
+  return Math.trunc(numberToTruncate * tenexp) / tenexp;
+};
+
 function App(): JSX.Element {
   const variableDefaults = {
     honey: 0,
     bees: 0,
-    costOfNextBee: 1
+    royalJelly: 0,
+    costOfNextBeeHoney: 1,
+    costOfNextBeeRoyalJelly: 0
   };
 
   // persistent variables
   const [honey, setHoney] = usePersistentState('honey', variableDefaults.honey);
   const [bees, setBees] = usePersistentState('bees', variableDefaults.bees);
+  const [royalJelly, setRoyalJelly] = usePersistentState(
+    'royalJelly',
+    variableDefaults.royalJelly
+  );
 
   // non-persistent varaibles (can be recalculated on page load)
-  const [costOfNextBee, setCostOfNextBee] = useState(
-    variableDefaults.costOfNextBee
+  const [costOfNextBeeHoney, setCostOfNextBeeHoney] = useState(
+    variableDefaults.costOfNextBeeHoney
+  );
+  const [costOfNextBeeRoyalJelly, setCostOfNextBeeRoyalJelly] = useState(
+    variableDefaults.costOfNextBeeRoyalJelly
   );
+  const [canBuyNextBee, setCanBuyNextBee] = useState(false);
```

Comment:
I like this, makes it easier if/when multiple components would need to check the same condition